src/electron/flux/action/device-actions.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Action } from 'common/flux/action';
+
+import { ConnectingPayload, ConnectionSucceedPayload } from './device-action-payloads';
+
+export class DeviceActions {
+    public readonly connectionSucceed = new Action<ConnectionSucceedPayload>();
```

Comment:
>connectionSucceed [](start = 20, length = 17)

The verb tenses here are inconsistent. I'd expect either connectionFailed and connectionSucceeded, or connectionFail and ConnectionSucceed.

Reply:
Good catch! Decided to go with the past tense.